src/components/DelegateStatement/CurrentDelegateStatement.tsx

```diff
@@ -103,6 +113,26 @@ export default function CurrentDelegateStatement() {
               }
             )?.topIssues
           : defaultIssues,
+
+      topStakeholders:
+        (
+          delegateStatement?.payload as {
+            topStakeholders: {
+              value: string;
+              type: string;
+            }[];
```

Comment:
is there anyway to type this better from wherever it is fetched? Having to add all of these `as` types makes it a bit tougher to read. Not a huge deal though, just a nit